src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1847,6 +1847,21 @@ private FlagsExpression<FileAttributes> FormatAttributeSwitchParameters()
         /// <param name="instance">Instance of PSObject wrapping a FileSystemInfo.</param>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2001:AvoidCallingProblematicMethods")]
         public static string Mode(PSObject instance)
+        {
+            return Mode(instance, includeHardLink: true);
+        }
+
+        /// <summary>
+        /// Provides a mode property for FileSystemInfo including HardLinks
+        /// </summary>
+        /// <param name="instance">Instance of PSObject wrapping a FileSystemInfo.</param>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2001:AvoidCallingProblematicMethods")]
+        public static string ModeWithoutHardlink(PSObject instance)
+        {
+            return Mode(instance, includeHardLink: false);
```

Comment:
The summary comment says `Provides a mode property for FileSystemInfo including HardLinks` but the code says `includeHardLink: false`, this is very confusing.

Reply:
Copy and paste - fixing.